From 873c0585710a375ef6749f1298a6a6830232c096 Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Sat, 22 Oct 2022 11:20:32 +0200
Subject: [PATCH 08/12] imx8mm_var_dart: allow cma address to be dynamically
 allocated

This avoids conflict with optee (32M) at 0x56000000.

Signed-off-by: Nate Drude <nate.d@variscite.com>
Signed-off-by: FrancescoFerraro <francesco.f@variscite.com>
(cherry picked from commit 4c62513edac1c7f5cf58e907f82d6a9d6fa57986)
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 include/configs/imx8mm_var_dart.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 44c97ca28a5..300da1d173f 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -139,7 +139,7 @@
 		"if test $sdram_size -le 512; then " \
 			"setenv cma_size cma=320M; " \
 		"else " \
-			"setenv cma_size cma=640M@1376M; " \
+			"setenv cma_size cma=640M; " \
 		"fi;\0" \
 	"mmcboot=echo Booting from mmc ...; " \
 		"run mmcargs; " \
-- 
2.34.1

